common/in_bounds.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Check if a point is within the bounds of e.g. the canvas.
+ * @param {number} x The x coordinate of the test point
+ * @param {number} y The y coordinate of the test point
+ * @param {number} width The width of the canvas
+ * @param {number} height The height of the canvas
+ * @returns true if (x, y) is within [0, width] x [0, height]
+ */
+export function in_bounds(x, y, width, height) {
```

Comment:
Make a sweep through the code and see where else this would be useful

Reply:
I checked, it's not as important for the other sketches to do bounds checking. For this one it is, because there's no meters to click outside the grid of the 16 of them.